filters/style_manager.py

```diff
@@ -123,7 +129,8 @@ def responseComplete(self):
             elif service == 'GetStyle':
                 return self.get_style(name, qgis_layer, request)
             elif service == 'SetDefaultStyle':
-                return self.set_default_style()
+                return self.set_default_style(
+                    name, project, project_path, qgis_layer, request)
```

Comment:
Perhaps you forgot this one? Just to remind you for the next PR.
The functionality itself is working.